src/style.css

```diff
@@ -1 +1,95 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+form {
+  width: 100%;
+  margin: 20px 0;
+}
+
+.search-bar {
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+  width: 100%;
+}
+
+.search-bar--big {
+  height: 70px;
+}
+
+.search-bar--small {
+  height: 42px;
+}
+
+.search-bar:hover {
+  box-shadow: 0 4px 4px 0 #00000040;
+  border-radius: 3px;
+}
+
+.search-bar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.search-bar_big:focus {
+  box-shadow: 0 4px 4px 0 #00000040;
+}
+
+.vector {
+  background-image: url(images/Search.svg);
+  background-color: white;
```

Comment:
This is default value, you can remove it

Reply:
done